deepchecks/vision/checks/data_integrity/image_property_outliers.py

```diff
@@ -47,11 +48,13 @@ class ImagePropertyOutliers(AbstractPropertyOutliers):
 
     def __init__(self,
                  image_properties: t.List[t.Dict[str, t.Any]] = None,
+                 property_input_type: PropertiesInputType = PropertiesInputType.IMAGES,
```

Comment:
Need to update also the docstrings in the inheriting checks. Missing docstring for this parameter